activesupport/lib/active_support/core_ext/hash/keys.rb

```diff
@@ -1,10 +1,14 @@
 class Hash
-  # Returns a new hash with all keys converted using the block operation.
+  # Returns a new hash with all keys converted using the +block+ operation.
   #
   #  hash = { name: 'Rob', age: '28' }
   #
-  #  hash.transform_keys{ |key| key.to_s.upcase }
-  #  # => {"NAME"=>"Rob", "AGE"=>"28"}
+  #  hash.transform_keys { |key| key.to_s.upcase }  # => {"NAME"=>"Rob", "AGE"=>"28"}
```

Comment:
There is a extra space before the `#` in all lines that you added. Could you remove them?